convert_v2.py

```diff
@@ -0,0 +1,110 @@
+import sys
```

Comment:
The file should be named convert_file.py according to the instructions

Reply:
I changed the name